MoviesMVP/MoviesMVP/Services/KeychainService.swift

```diff
@@ -0,0 +1,39 @@
+// KeychainService.swift
+// Copyright © RoadMap. All rights reserved.
+
+import KeychainAccess
+
+final class KeychainService {
```

Comment:
```suggestion
final class KeychainService {
```
Нарушаем букву D из Solid если не знаешь что делать спроси у Дима C или Эрнеста я им объяснял утром